readabilitySAX.js

```diff
@@ -438,7 +438,7 @@ Readability.prototype.onclosetag = function(tagName){
 	if(this._settings.searchFurtherPages && tagName === "a"){
 		this._scanLink(elem);
 	}
-	else if(tagName === "title"){
+	else if(tagName === "title" && tagName in headerTags){
```

Comment:
This will never be true. A better check would be if this is the first title element: `&& !this._origTitle`.